### PR TITLE
Add template for onestop_english dataset

### PR DESCRIPTION
**Pull request by trishalaneeraj**

### CODE REVIEW

File: templates/onestop_english/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+      How would you rate the difficulty level of the passage above for automatic readability
+      assessment? {{"ele"}}, {{"int"}} or {{"adv"}} level?
+
+
+      |||
+
+
+      {{ ["ele", "int", "adv"] [label] }}'
```
Comment (VictorSanh) [inline templates/onestop_english/templates.yaml:31]: replace "ele" by "elementary", "int" by "intermediate", etc. in both the prompt and the label

File: templates/onestop_english/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    name: esl_variation
+    reference: ''
+    task_template: true
+  e1289be8-7e81-4a85-bfb3-225fd31749a7: !Template
+    id: e1289be8-7e81-4a85-bfb3-225fd31749a7
+    jinja: 'How would you assess the reading difficulty of the text passage below?
+      Choose from one of {{"ele"}}, {{"int"}} or {{"adv"}} levels?
```
Comment (VictorSanh) [inline templates/onestop_english/templates.yaml:57]: no need for `?`

File: templates/onestop_english/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+      {{ ["ele", "int", "adv"] [label] }}'
+    name: ara_context
+    reference: ''
+    task_template: true
+  de75ccb8-c0ba-4510-abf8-649b42019cd5: !Template
+    id: de75ccb8-c0ba-4510-abf8-649b42019cd5
+    jinja: 'If the text passage below was to be presented to someone learning English
+      as their second language, how are they likely to rate the difficulty level of
```
Comment (VictorSanh) [inline templates/onestop_english/templates.yaml:38]: how are they likely **going**

File: templates/onestop_english/templates.yaml (hunk 1 above)
Comment (VictorSanh) [reply templates/onestop_english/templates.yaml:31]: same for all templates

**Review by VictorSanh — CHANGES_REQUESTED** [review]
Thank you @trishalaneeraj !

a few comments, please ping me when you addressed them, I'll merge then!

**Comment by trishalaneeraj** [comment]
@VictorSanh Thanks! I've addressed the comments above.

**Review by VictorSanh — APPROVED** [review]
thank you! please update the spreadsheet
merging now

**Pull request merged**